pkg/cloudprovider/types/types.go

```diff
@@ -76,6 +76,12 @@ type Provider interface {
 	SetMetricsForMachines(machines clusterv1alpha1.MachineList) error
 }
 
+// NetworkConfig holds information about cluster networking.
+// PodCIDRs fields is used to choose IPv4, IPv6 or dual-stack modes. 
```

Comment:
Move the `PodCIDRs` comment from here to one line above the `PodCIDRs` struct member.